common/compositor/gl/glprogram.cpp

```diff
@@ -351,6 +355,16 @@ void GLProgram::UseProgram(const RenderState &state, GLuint viewport_width,
                        src.texture_matrix_);
     glActiveTexture(GL_TEXTURE0 + src_index);
     glBindTexture(GL_TEXTURE_EXTERNAL_OES, src.handle_);
+    if (src.is_solid_color_){
```

Comment:
it seems does not need this if judge  here.
The logic is that all layers need to send this glUniform4f to set the color
For normal layer the color is 0, 0, 0, 0
For Solid color the color is what sf set.
We can make the code more simple